colour/io/luts/sony_spi3d.py

```diff
@@ -82,8 +82,8 @@ def read_LUT_SonySPI3D(path):
 
                 size = DEFAULT_INT_DTYPE(tokens[0])
             if len(tokens) == 6:
-                indexes.append(parse_array(tokens[:3]))
-                table.append(parse_array(tokens[3:]))
+                indexes.append(as_float_array(tokens[:3]))
+                table.append(as_float_array(tokens[3:]))
```

Comment:
Careful here! The indexes/indices should be of type int, so you should use `as_int_array`:

```python
indexes.append(as_int_array(tokens[:3]))
```

Reply:
OK missed that, will make the change ASAP!